testutil/log.go

```diff
@@ -0,0 +1,29 @@
+package testutil
+
+import (
+	"fmt"
+	"reflect"
+)
+
+func Log(tag string, message ...interface{}) {
+	if len(message) > 0 {
+		log(tag, message[0])
+	} else {
+		log(tag, nil)
+	}
+}
+
+func log(tag string, a interface{}) {
+	if a == nil {
+		fmt.Println(tag)
+		return
+	}
+
+	// a!=nil
+	array := reflect.ValueOf(a)
+	if array.Kind() == reflect.Array || array.Kind() == reflect.Slice {
+		fmt.Println(ArrayToString(tag, a))
```

Comment:
Using `fmt.Println` while running tests is no good.  This is why I complained last time about not being able to have access to `testing.T` in these functions -- you need to be doing `t.Logf` so that you get it logged to the console with `go test -v`.